Silicon/NXP/iMX6Pkg/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.h

```diff
@@ -41,4 +41,6 @@ typedef struct {
   UINT8 MacAddress[6];
 } GLOBAL_PAGE_DATA;
 
+extern EFI_GUID gProvisioningGuid;

```

Comment:
ProvisioningGuid might be too generic. It'll be okay for now but we'll probably have to scope the name before it goes into mainline. Something like giMXPlatformProvisioningGuid or similar...

Reply:
Updated with giMXPlatformProvisioningGuid